components/Rainbowkit/Tally/Tally.ts

```diff
@@ -0,0 +1,29 @@
+import { Chain, Wallet } from '@rainbow-me/rainbowkit';
+import { InjectedConnector } from 'wagmi/connectors/injected';
+
+import TallyIcon from 'assets/png/rainbowkit/tallyho.png';
+
+type TallyOptions = {
+	chains: Chain[];
+	shimDisconnect?: boolean;
+};
+
+const Tally = ({ chains, shimDisconnect }: TallyOptions): Wallet => ({
+	id: 'tally',
+	iconBackground: '#FFF',
+	name: 'Tally Ho',
+	iconUrl: async () => TallyIcon,
+	downloadUrls: {
+		android: 'https://play.google.com/store/apps/details?id=io.gnosis.Tally',
+		ios: 'https://apps.apple.com/us/app/gnosis-Tally/idid1515759131',
```

Comment:
These links don't work, should probably set them to empty strings as Tally does not have Android/iOS apps?

Reply:
Yes. Good catch. I will update these links.

Reply:
I pushed a commit to remove these links

Reply:
@Tburm I added the link to `Tally Ho`'s chrome browser extension.  
![image](https://user-images.githubusercontent.com/4819006/191329242-398f3c83-d573-4cd9-84c3-7781013859b9.png)